### PR TITLE
refactor: config_files is a readonly config

### PR DESCRIPTION
**Pull request by zmstone**
the `node.config_files` config is only used to map to app env,
but a user can never change it because EMQX has to boot from `etc/emqx.conf`

### CODE REVIEW

**Review by zhongwencool — APPROVED** [review]

**Review by zhongwencool — APPROVED** [review]

**Review by ieQu1 — DISMISSED** [review]

File: changes/v5.0.15/fix-9750.en.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Reload overriding configs after boot.
+Prior to this change, two configs were allow to change from dashboard, but will not take effect after reboot:
```
Comment (thalesmg) [inline changes/v5.0.15/fix-9750.en.md:2]:
```suggestion
Prior to this change, two configs were allowed to be changed from dashboard, but would not take effect after reboot:
```

**Review by thalesmg — APPROVED** [review]
Q: this seems to be a doc and test only change.  Do we need changelogs for this?

**Comment by zmstone** [comment]
> Q: this seems to be a doc and test only change. Do we need changelogs for this?

the change log is not for this pr

**Review by thalesmg — APPROVED** [review]

**Comment by zmstone** [comment]
Sorry, the most important commit was lost during rebase...
Now just pushed after rebased again.

**Review by lafirest — APPROVED** [review]

**Review by zhongwencool — APPROVED** [review]

**Pull request merged**